.github/workflows/run-precommit.yaml

```diff
@@ -19,11 +19,7 @@ jobs:
       - name: Checkout repository 🔔
         uses: actions/checkout@v3
 
-      # https://github.com/pre-commit/action to enable cache
-      - uses: pre-commit/action@v3.0.0
-
-      - name: Install pre-commit 📦
-        run: pip install pre-commit
-
       - name: Run terraform pre-commit ⚡️
-        run: pre-commit run terraform_fmt
+        uses: pre-commit/action@v3.0.0
+        with:
+          extra_args: --all-files terraform_fmt
```

Comment:
Seems like this is only running pre-commit for terraform, not for rest of the code.

Like we did earlier: https://github.com/nebari-dev/nebari/actions/runs/4873704426/jobs/8693649224#step:3:63

```bash
Run pre-commit run --show-diff-on-failure --color=always --all-files
fix end of files.........................................................Passed
trim trailing whitespace.................................................Passed
check json...............................................................Passed
check yaml...............................................................Passed
check toml...............................................................Passed
check that executables have shebangs.....................................Passed
codespell................................................................Passed
black....................................................................Passed
ruff.....................................................................Passed
isort....................................................................Passed
Terraform fmt............................................................Passed
mdformat.................................................................Passed

```

Reply:
Not sure what you are saying here. We don't want to run all pre-commit hooks here. This is what we have [pre-commit CI](https://results.pre-commit.ci/run/github/252841745/1683111024.eUdbjnNgTrW3HAHR79urTQ) for. It was introduced in #1602 and runs everything but `terraform_fmt`:

https://github.com/nebari-dev/nebari/blob/7653c6700a2bfdfb4e4c07e2e869618663dfe96f/.pre-commit-config.yaml#L15-L20

The reason for this split setup is that `terraform_fmt` requires `terraform` to be available on the system and that is not possible on pre-commit CI.

Reply:
Ah, yes. I didn't realise that. Thanks for explaining.